WTWSMS/events/ze_lord_spiritual_events.txt

```diff
@@ -29,7 +29,6 @@ character_event = {
 			independent = yes
 			higher_tier_than = DUKE
 		}
-		proselyte_religion_trigger = yes
```

Comment:
If you find the core features of the mod to be "crap", you are free to ignore them andby consequence the mod, but don't come here and change things you wouldn't include in the first place and thus don't understand the purpose of. Proselytism both represents the likelihood of sending missionaries inside and outside the realm. I'm going to change that description right away.